package.json

```diff
@@ -11,6 +11,7 @@
     "test:spec": "nyc --report-dir ./reports/coverage mocha --require ts-node/register --reporter spec 'spec/**/*.spec.ts'",
     "test:e2e:protractor": "protractor e2e/protractor/protractor.conf.js",
     "test:e2e:webdriverio": "wdio e2e/webdriverio/wdio.conf.js",
+    "test:e2e:puppeteer": "ts-mocha -p tsconfig.json e2e/puppeteer/chrome-authenticator-extension.spec.ts",
```

Comment:
Let's not introduce `ts-mocha` in this PR and instead use `mocha --require ts-node/register --reporter spec e2e/puppeteer/chrome-authenticator-extension.spec.ts`

Since mocha is used for both unit tests and Puppeteer, I think it makes sense to have a separate PR that migrates both things to a new tool.

Reply:
de-boarded ts-mocha